### DISCUSSION TITLE
Balance won't load

### DISCUSSION THREAD

**Discussion by nonswad** [phantom/docs#245] [Help & Support]
### Summary

Phantom wallet showing me the message of  "We are having trouble updating networks, some of your token balances and prices may be out of date. Your funds are safe" and not showing me the coins or anything

### Steps to Reproduce

Login to your wallet extension and open it.

### Phantom Version

V 24.9.1

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@nonswad For support with this issue and any other issues using Phantom please contact Phantom support from your phantom wallet in Settings > Help & Support.

This discussion board is for developer questions only. Thank you for understanding!
